frappe/core/doctype/log_settings/log_settings.py

```diff
@@ -10,8 +10,10 @@
 
 
 class LogSettings(Document):
-	def clear_logs(self):
+	def clear_logs(self, commit=False):
 		self.clear_email_queue()
+		if commit:
+			frappe.db.commit()
```

Comment:
Just this commit is required, error log is myisam and activity log is last table in list so commit happens from bg job completion anyway.

Reply:
DOne